File: mobile/ios/src/data/models/order.swift
import Foundation

struct Order: Codable {
  let orderLines: [OrderLine]
  let paymentProcessorReference: String?
  let paymentVerificationReference: String?
  let nonce: String
  let orderNotes: String?
  let servingType: VenueServingType
  let table: String?
}

File: mobile/android/app/src/main/java/io/meshtech/koasta/net/model/Order.kt
package io.meshtech.koasta.net.model

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Order (
  var orderLines: List<OrderLine>,
  var paymentProcessorReference: String?,
  var paymentVerificationReference: String?,
  var nonce: String,
  var orderNotes: String?,
  var servingType: Int,
  var table: String?
)
